chapter_1/exercise_1_12/copy_io_nl.c

```diff
@@ -1,18 +1,18 @@
-#include <stdio.h>                                                               
-                                                                                 
-int main()                                                                       
-{                                                                                
-        char c;                                                                  
-        char last_char = -1;                                                     
-        while ((c = getchar()) != EOF) {                                         
-                if(c==' ' || c=='\t' ||  c=='\n'){                               
-                        if(last_char!=' ' &&  last_char!='\t' && last_char!='\n'){
-                                putchar('\n');                                   
-                        }                                                        
-                }                                                                
-                else {                                                           
-                        putchar(c);                                              
-                }       
-                last_char=c;
-        }                                                                        
-}        
+#include <stdio.h>
+
+int main()
+{
+  char c;
+  char last_char = -1;
```

Comment:
Please add `\n` before the while and after the while block. Each logical block should be separated by a new line